python/packages/nisar/products/insar/InSAR_base_writer.py

```diff
@@ -361,12 +361,21 @@ def add_RSLC_to_procinfo_params_group(self, rslc_name: str):
                                   ' composite of data collected in multiple'
                                   ' radar modes, "False" otherwise')
         ds_params = [
+            DatasetParams(
+                "rfiMitigation",
+                rfi_mitigation,
```

Comment:
```suggestion
                np.bytes_(rfi_mitigation),
```
Don't you need to convert it to a byte string?